src/test/java/org/folio/marc4ld/mapper/field775/Bibframe2Marc775IT.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.marc4ld.mapper.field775;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.folio.marc4ld.mapper.test.MonographTestUtil.getSampleInstanceWithWork;
+import static org.folio.marc4ld.mapper.test.MonographTestUtil.getSampleWork;
+import static org.folio.marc4ld.mapper.test.TestUtil.loadResourceAsString;
+
+import org.folio.marc4ld.mapper.test.SpringTestConfig;
+import org.folio.marc4ld.service.ld2marc.Bibframe2MarcMapperImpl;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@EnableConfigurationProperties
+@SpringBootTest(classes = SpringTestConfig.class)
+class Bibframe2Marc775IT {
```

Comment:
We should use "ld" term instead of "bibframe" at least for new classes. Old ones will be also renamed at some point of time, perhaps we should create a tech debt card. @pkjacob What do you think?

Reply:
Yes. I will create a tech debt ticket.

Reply:
Done